### PR TITLE
Monitoring health check support

### PR DESCRIPTION
**Pull request by bound2**
Add `/health/liveness` endpoint to probe whether service is healthy or not. As of right now it only checks for database availability.

### CODE REVIEW

File: server/src/server.ts [hunk 1]
```diff
@@ -35,6 +36,7 @@ const io = new Server(server, {
 });
 const protection = csurf({ cookie: true });
 app.use("/api/v1", api, protection);
+app.use("", monitoringApi);
```
Comment (Dev-CasperTheGhost) [inline server/src/server.ts:39]: You can remove the `"",` from here

**Review by Dev-CasperTheGhost — CHANGES_REQUESTED** [review]
All looks good, just fix that minor request change

**Comment by Dev-CasperTheGhost** [comment]
This was tested correct?

**Comment by bound2** [comment]
Yep. Try it out. Run it with docker-compose go check http://localhost:3030/health/liveness. Then stop db container from Docker Desktop, try the same url again and it should show you "status": "DOWN". In addition to that you'll see errors in console as docker-compose is doing health check periodically.

Should probably package `api.ts` and `monitoring-api.ts` into api folder, but import from 'api/api' seems a bit weird (should probably rename api.ts to something else, but I couldn't come up with a good name). Maybe you can think of something.

**Comment by Dev-CasperTheGhost** [comment]
Alright, nice.

**Review by Dev-CasperTheGhost — APPROVED** [review]
Looks good now!

**Pull request merged**